corehq/apps/geospatial/static/geospatial/js/case_grouping_map.js

```diff
@@ -7,21 +7,28 @@ hqDefine("geospatial/js/case_grouping_map",[
     $,
     ko,
     _,
-    initialPageData
+    initialPageData,
```

Comment:
Should this comma be here? I know I've had issues in the past with a `Parsing error: unexpected token` error when a comma was put at the end of the last item.

Reply:
Let me remove it rather then